### PR TITLE
feat(scripts): Add create local database

### PR DESCRIPTION
**Pull request by Baboo7**

### CODE REVIEW

File: carbon_bombs/io/neo4j.py [hunk 1]
```diff
@@ -435,3 +436,20 @@ def delete_all(tx):
...
+
+    # Define dict to iterate over three types node creation
+    dict_nodes = {
+        "carbon_bomb": carbon_bombs.to_dict(orient='records'),
+        "company": companies.to_dict(orient='records'),
+        "bank": banks.to_dict(orient='records'),
+        "country": countries.to_dict(orient='records'),
+    }
+
+    with open('database.json', 'w') as convert_file:
```
Comment (Nathanlauga) [inline carbon_bombs/io/neo4j.py:454]: Put the file name in `conf.py`
Also maybe use : the following path `{DATA_NEO4J_PATH}/database.json`

**Review by Nathanlauga — CHANGES_REQUESTED** [review]
Need to put the database file name into conf.py
Also do we need to push this file into the repo ? if not then we should put it into .gitignore

**Comment by Baboo7** [comment]
@Nathanlauga it's updated!

**Review by Nathanlauga — APPROVED** [review]
Ok!

**Pull request merged**